.github/workflows/ci.yml

```diff
@@ -10,23 +10,17 @@ on:
 jobs:
   test:
     runs-on: ubuntu-latest
-    container: ${{matrix.image}}
     strategy:
       fail-fast: false
       matrix:
-        image:
-          - ruby:2.5.0
```

Comment:
I appreciate your enthusiasm about TruffleRuby but I'd like to keep our current set of rubies as well.

Reply:
I will revert it back, just want to skip all other ruvies for now

Reply:
OK. Let me know when this PR is ready.